src/Extension/LoginAttemptNotifications.php

```diff
@@ -89,7 +89,7 @@ public function init()
             }
         }
 
-        $session->set('LoginAttemptNotifications.SessionLastVisited', DBDatetime::now()->Format('Y-m-d H:i:s'));
+        $session->set('LoginAttemptNotifications.SessionLastVisited', DBDatetime::now()->Format('y-MM-dd HH:mm:ss'));
```

Comment:
you can use `DBDatetime::ISO_DATETIME` as the format argument

Reply:
True!